nx.json

```diff
@@ -34,7 +34,8 @@
     },
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32393&commitId=7cce03d69efffb7dd21363a45512ad0286d77f38&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|21| Changed|
|Avatar Converged.badgeMask.chromium.png|20| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|4| Changed|
|Drawer.Full Overlay High Contrast.chromium.png|2239| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>